packages/workgrid-client/src/client.test.ts

```diff
@@ -204,6 +237,63 @@ describe('@workgrid/client', () => {
     `)
   })
 
+  test('graphql', async () => {
+    const query = `
```

Comment:
If you tag this with gql, most editors will add syntax highlighting and prettier will auto format it.

Option 1
```suggestion
    const gql = String.raw

    const query = gql`
```

Option 2
```suggestion
    const query = /* graphql */ `
```